### DISCUSSION TITLE
Merge ➕ Connect: Your November Pull Requests Check-In

### DISCUSSION THREAD

**Discussion by ghostinhershell** [community/community#144017] [Pull Requests]
Hey hey! 👋 Reporting in for a new Pull Requests Community Check-In 🗞️

![Cartoon character holding a coffee cup stands on purple clouds next to a rocket in space. Text reads "Pull Requests Check-in." Planets and stars in the background. GitHub logo in the corner.](https://github.com/user-attachments/assets/471578c6-6b8f-49c0-bfce-1a13f74cbd13)

## 🏆​​ Top Contributors in the Community
Big Kudos to these Superstar Community Contributors 👏
- Shout out to @mehulmina064 for helping a new open source contributor understand best practices for contributing: [Should I Open an Issue or Raise a PR First?](https://github.com/orgs/community/discussions/140951)
- Thank you @Berchez for lending a helping hand to a user who was having issues applying a custom font to their project: [Need help for fixing the font style](https://github.com/orgs/community/discussions/141608)
- Thank you @fxrdhan for helping us understand the [Impact of Force-Pushing on Existing Pull Requests](https://github.com/orgs/community/discussions/142466)

## 🧠 Popular Unanswered Posts in the Community
Got your thinking cap on? These popular community posts could use your expertise!
- [Empty pull request from fork's main to main flagged as "auto merged"](https://github.com/orgs/community/discussions/143026)
- [Merge Strategies](https://github.com/orgs/community/discussions/140749)
- [Find All the PRs i need go handle](https://github.com/orgs/community/discussions/141407)
- [[error]fatal: could not read Username for 'https://': terminal prompts disabled](https://github.com/orgs/community/discussions/142382)

**Have you been helped by a community superstar that you’d like to see celebrated?** Tag them to nominate them below!

For community members who post questions, [marking responses that solves your question as the answer](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#marking-a-comment-as-an-answer) goes a long way. When you mark a comment in the Discussion as an answer, GitHub will highlight the comment to help other community members quickly find the answer as well.
<img src="https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2" alt="The image is a screenshot showing where users can click to mark a response as the answer" width="800" height="300">

Remember to thank these and other community members that take the time to comment and help troubleshoot these problems. An acknowledgement of a job well done can have an incredible effect and make someone’s day. ☀️ We’re here to learn, discover and connect so try and mark a comment as an answer if suitable, give a thank you and select a reaction as a way to show your appreciation! 

## 🆕 What’s New With Pull Requests?
Looking to keep up with the latest Pull Requests? We’ve got you covered! This list brings together the most recent PR releases, including new features, fixes, and improvements that will streamline workflows, enhance collaboration, and boost productivity. There's something here for every team member and contributor. Dive in and see what’s new in the world of Pull Requests!
- [[Public Preview] Refine and validate code review suggestions with Copilot Workspace](https://github.com/orgs/community/discussions/142278)
- [[Public Beta] Pull Request Commits Page Refresh](https://github.com/orgs/community/discussions/137725) 
- [Push rules public beta](https://github.com/orgs/community/discussions/118843)
- [Introducing Enhanced Code Review on GitHub Mobile](https://github.blog/changelog/2024-04-09-introducing-enhanced-code-review-on-github-mobile/) 

## ✍️ Quote of the Month
_"Don’t be afraid to publish along the way. You don’t have to wait until you’re done to drop a perfect, finished artifact from the sky (in fact, you may use that as an excuse to never publish)."_
> from [**Publishing Your Work Increases Your Luck**](https://github.com/readme/guides/publishing-your-work) by Aaron Francis // Marketing Engineer, Tuple

— 
A very big thank you to everyone in the community who actively contributes thoughts and answers to the questions that come into the Pull Requests category!

**Comment by madisonscott** [comment]
@ghostinhershell thanks for all the work you've all been doing in the PR space! Wondering if the team might consider taking this on soon? 🙏  https://github.com/orgs/community/discussions/65597
